departments/development/events/Meeting/20240416-Meeting-Internal-Minutes.md

```diff
@@ -2,15 +2,108 @@
+    - 🤝 **Vesting** 🤝
+        - Unstaking on April 19th
+        - Send back funds that exceed Q2 Budget
+    - 🤝 **Github & Asset Ownership** 🤝
+        - No Updates
+        - Created a Policy for Meeting Minutes
+    - 🤝 **Critical Infrastructure** 🤝
+        - RFPs are published
+        - No emails received
+        - Discussion on the RFPs
+          - Investigating whether the RFP lack of response is a reach issue with Communications
+        - [ ] Take up Comm Engagement with Council Meeting
+    - 🤝 **Budget & Treasury** 🤝
+        - Proposal to pass Q2 Budget at next Council Meeting
+          - Should include transfer of balance from Council to Department
+          - Dev Dept needs to send back funds that exceed Q2 Budget
+    - 🤝 **Department Policies** 🤝
+      - Created github structure for policies and procedures
+      - People can use github to submit policy and procedures as a branch to review
+      - Codespaces & HackMD can be used as a frontend to submit policies
+    - 🤝 **Conflict of Interest Policy** 🤝
+      - Discussion on the policy point 1.3
+        - discussion over MCap v % of Total Supply or Trading Volume
+      - Minor changes to the policy
+      - Macks brought up COI Policy at Operations Meeting
+        - Will Discuss at next Operations Meeting
+    - 🤝 **Development Relations** 🤝
+      - No updates
+    - 🤝 **Internal Tooling** 🤝
+      - @kopeboy gave a presentation on Monday about the tooling and what he has been working on
+      - Andre:
+        - PRO:  It's useful and the work is good
+        - CON:  The cost to maintain it
+      - @kopeboy:
+        - Still experimenting with the tooling
+        - No specific updates now
+        - Created:
+          - data structure & content types for meetings, working groups with issues and tasks
+          - Event Types
+          - Department member roles
+          - Automatic workflows
+        - Meeting scheduled for Demo at 16UTC on April 22nd 2024
+
+1. Review responses to Dev Department Contact Form
+  - no new responses
+
+1. Darkside Podcast Response
+  - Lorenzo is still working on a response
+
+1. New Metric Public Presentation
+  - Meeting Scheduling for Thursday 18th April 2024 at 17UTC
+
+1. Documentation
+  - getting ownership of the documentation
+  - discussion about how to move forward
+  - bring to council to raise awareness of work required on documentation
+
+1. Juno.tools
+  - waiting on Q2 Budget approval
+
+1. #cross-department
+  1. Interchain-Test update request
+
+1. Council Meeting Agenda
+  1. Budget Submission Propsal for Q2
+  1. Progress reports on Working Groups: 
+    1. Abstract SDK
+    1. Policy
+    1. Asset Management
+    1. Security
+    1. Treasury / Reserve
+1. Abstract Money Market dApp would need to be developed
+  1. Discussion on the dApp and that it needs to be created
+  1. Money Market dApp and waiting for feedback from Abstract
+  1. Private TG Group
+  1. @kopeboy: would like to define a strategy
```

Comment:
`and thinks what's being discussed in private has strategic relevance since our resources are very limited, so he would like to have access or stay informed.`

Reply:
incorporated into minutes